File: src/main/java/ru/hubsmc/hubslobby/listener/SuccessLoginEvent.java
package ru.hubsmc.hubslobby.listener;

import fr.xephi.authme.events.LoginEvent;
import fr.xephi.authme.events.LogoutEvent;
import org.bukkit.event.EventHandler;
import org.bukkit.event.Listener;
import ru.hubsmc.hubscore.listener.JoinEvent;

public class SuccessLoginEvent implements Listener {

    @EventHandler
    public void onLogin(LoginEvent event) {
        JoinEvent.onJoinOrLogin(event.getPlayer());
    }

    @EventHandler
    public void onLogout(LogoutEvent event) {
        event.getPlayer().kickPlayer("LOGOUT IS NOT ALLOWED HERE!");
    }

}

File: src/main/java/su/hubs/hubslobby/listener/SuccessLoginEvent.kt
package su.hubs.hubslobby.listener

import fr.xephi.authme.events.LoginEvent
import fr.xephi.authme.events.LogoutEvent
import org.bukkit.event.EventHandler
import org.bukkit.event.Listener
import su.hubs.hubscore.listener.JoinEvent

class SuccessLoginEvent : Listener {
    @EventHandler
    fun onLogin(event: LoginEvent) {
        JoinEvent.onJoinOrLogin(event.player)
    }

    @EventHandler
    fun onLogout(event: LogoutEvent) {
        event.player.kickPlayer("LOGOUT IS NOT ALLOWED HERE!")
    }
}
